tiledb/sm/cpp_api/array.h

```diff
@@ -643,6 +643,48 @@ class Array {
         config ? config->ptr().get() : nullptr));
   }
 
+  /**
+   * @brief Consolidates the fragments with the input uris into a single
+   * fragment.
+   *
+   * You must first finalize all queries to the array before consolidation can
+   * begin (as consolidation temporarily acquires an exclusive lock on the
+   * array).
+   *
+   * **Example:**
+   * @code{.cpp}
+   * const char* fragment_uris[2] = {
+   * "hdfs:///temp/my_array/__fragments/1",
+   * "hdfs:///temp/my_array/__fragments/2"};
+   *
+   * tiledb::Array::consolidate(
+   *     ctx,
+   *     "s3://bucket-name/array-name",
+   *      fragment_uris,
+   *      2,
+   *      config);
+   * @endcode
+   *
+   * @param ctx TileDB context
+   * @param array_uri The URI of the TileDB array to be consolidated.
+   * @param fragment_uris The input fragment uris.
```

Comment:
Adjust this to reflect what is in tiledb_array_consolidate_fragments

Reply:
They match with the params in array::consolidate which is the method they are for. I only changed the name ```uri``` to ```array_uri```...

Reply:
Rebased against dev and matched the descriptions